docs/markdown/snippets/python_module_env.md

```diff
@@ -0,0 +1,8 @@
+## New option to choose python installation environment
+
+It is now possible to specify `-Dpython.install_env` and choose how python modules are installed.
+
+- `venv`: assume that a virtualenv is active and install to that
+- `system`: install to the global site-packages
```

Comment:
This may be hard to understand (it could be the conda/homebrew/etc one or the actual system provided one - you mean the former here). How about something like _"install to the site-packages of the active Python interpreter (if a virtualenv, this means the Python it is derived from)"_?

Reply:
I think maybe I was thinking of that when I used the word "global", which has the connotation, to me, of "non-venv" but still within the interpreter.

What about this change?

```diff
-- `system`: install to the global site-packages
+- `system`: install to the global site-packages of the selected interpreter
+  (the one that the venv module calls --system-site-packages)
```

Reply:
That looks good, it's quite clear.